src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -266,5 +267,16 @@ public void Build(IListedCapabilityStatement statement)
                 statement.TryAddRestInteraction(SystemRestfulInteraction.Transaction);
             }
         }
+
+        public ITransactionScope BeginTransaction()
+        {
+            _transactionScope = new DefaultTransactionScope();
```

Comment:
Does SqlServerFhirDataStore need to keep a reference to the transaction in this case? I feel that it is creating and assigning ownership to the calling component. Thoughts?